### ISSUE TITLE
Crash when using pipette

### ISSUE THREAD

**Issue by rhean** [robot256/MultipleUnitTrainControl#19]
When right-clicking I get this error, game version 2.0.12
Edit: clicking Q to use the pipette.

```
The mod Multiple Unit Train Control (1.0.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event MultipleUnitTrainControl::on_player_pipette (ID 105)
LuaItemStack API call when LuaItemStack was invalid for read.
stack traceback:
	[C]: in function '__index'
	__Robot256Lib__/script/blueprint_replacement.lua:113: in function 'mapPipette'
	__MultipleUnitTrainControl__/control.lua:565: in function <__MultipleUnitTrainControl__/control.lua:564>
```

**Comment by robot256** [comment]
Thanks for the report. This is actually a big in my library and it happened with another mod too. What were you doing when it happened?  Can you reproduce this crash, and can you send me the log file and a save that does it? There might be a bug in the API function, or an incorrect error message being produced by the game, or there might be out-of-date mods involved.

**Comment by rhean** [comment]
I have to correct myself from the first post, its when clicking Q to use the pipette. I loaded up all compatible saves I have with only the base game and everything else disabled except the required libs. The exact same thing happens with all the saves I have, I'll attach a log of loading a game with everything else disabled.

```
43.744 Error MainLoop.cpp:1429: Exception at tick 10338076: The mod Multiple Unit Train Control (1.0.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event MultipleUnitTrainControl::on_player_pipette (ID 105)
LuaItemStack API call when LuaItemStack was invalid for read.
stack traceback:
	[C]: in function '__index'
	__Robot256Lib__/script/blueprint_replacement.lua:113: in function 'mapPipette'
	__MultipleUnitTrainControl__/control.lua:565: in function <__MultipleUnitTrainControl__/control.lua:564>
  45.680 Quitting: user-quit.
```

[factorio-current.log](https://github.com/user-attachments/files/17561783/factorio-current.log)

**Comment by robot256** [comment]
Where was your cursor when you pressed Q to pipette?

**Comment by rhean** [comment]
Over a MU locomotive, showing the bounding-box the unit-info in the right panel (no problem with regular locomotives).

**Comment by robot256** [comment]
Fixed in the release 1.0.1
